### ISSUE TITLE
"Automatically merge series across multiple folders" library option makes certain items inaccessible in User Views.

### ISSUE THREAD

**Issue by Jaxzog** [Aanok/jftui#28]
**jftui version**
jftui 0.5.2, libmpv 1.109, libcurl 7.83.1 OpenSSL/1.1.1o, yajl 20100

**Describe the bug**
"Automatically merge series across multiple folders" library option makes certain items inaccessible in User Views.

**To Reproduce**
turn on "Automatically merge series across multiple folders" in jellyfin under manage library. Have a series with multiple seasons in different folders that merge into one in the web ui. In jftui it will only show one series entry, like in the web ui, but if you open that entry it will only show the contents from one of the folders rather than both. This makes the contents from the other folder inaccessible.

**Expected behavior**
Merged series show the contents from both merged folders rather than only one.

**Comment by Aanok** [comment]
Hi! I don't use this feature so I'm not familiar with it. Would you mind describing how it looks like on the filesystem with an example? In particular, can you have the same season across distinct folders or must distinct seasons be in distinct folders?

**Comment by Jaxzog** [comment]
> In particular, can you have the same season across distinct folders or must distinct seasons be in distinct folders?

If I'm being perfectly honest, I'm entirely not sure. the reason I only came across this bug now rather than earlier is because it rarely comes into effect.

At least in my case, I have two seasons of one show in two different directories on my filesystem because they are from different sources (bluray,download,etc.). The layout in the filesystem is much the same as if they were two distinct shows, it is merely the metadata in jellyfin that merges them.
 Once the metadata is entered and jellyfin recognizes that they are 2 seasons of 1 show rather than 2 distinct shows, it merges the menu entries into one show entry in the shows view. When you enter that one merged menu entry, where you would normally see the "season _" "specials" etc. entries, you see entries from both.

now that I think about it, the specials folder from both seasons was merged into one, so maybe it is possible have episodes from a single season in different directories, although I'm not sure in what case one would ever want to do that.

(I apologize for not being very helpful, as I do not know all that much about the feature myself either)

Also I would just like to put that it is not that big of a deal given that I can just turn this option off and access each season independently, and all works perfectly fine.
 But some indiviuals may use jftui and yet not have administrator access to the jellyfin server, or may be unable to change the options due to other people using it, so optimally it would get fixed for those people.
